01-JS-DOM-APIs/Ex04/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+		<meta charset="UTF-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+   	<title> JS 04 </title>
+   	<meta name="description" content="Die promises, die">
+   	<meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Bubbler+One" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+     	<h1>This it my fourth JS excercise!</h1>
+    </header>
+    <div class="repWrapper">
+      <h2>Repositories:</h2>
+      <ul id="repoList">
+      </ul>
+    </div>
+    <script>
+      function getRep() {
+        let repRequest = new XMLHttpRequest();
+        repRequest.open ('GET', 'https://api.github.com/search/repositories?q=JavaScript', true);
+        repRequest.send ();
```

Comment:
Let's hold on this one. Once you finished the exercise 3, we go a head an review the 4. You will need to add the reusable function to make Ajax calls here.